lib/payoneer.rb

```diff
@@ -43,17 +43,15 @@ def self.make_api_request(method_name, params = {})
     inner_content
   end
 
-  private
-
-  def self._configuration
-    self.configuration ||= Configuration.new
+  def self.configuration
+    self._configuration ||= Configuration.new
```

Comment:
Why not use a class instance variable `@configuration` and why make `_configuration` a public accessor? Seems like this method should be more than enough and if an external thing needs access to set the instance variable it would go through a `configuration=` setter.

`_prefixes` in Ruby are a sure code-smell (at least to me).